subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -415,6 +418,8 @@ int fota_download_start_with_image_type(const char *host, const char *file,
 	const enum dfu_target_image_type expected_type)
 {
 	static int sec_tag_list[1];
+	static uint32_t host_sha256;
+	static uint32_t file_sha256;
```

Comment:
Why these are `static`?
They don't have to be.

Reply:
Fixed